app/scripts/proactive/view/utils/undo.js

```diff
@@ -37,17 +37,28 @@ define(
                 activeTask: StudioApp.views.workflowView.getActiveTask()};
             this._saveIfDifferent(state);
         },
-        undo: function () {
+        undo: function (num) {
             if (undoStates.length <= 1) {
                 StudioClient.alert("No further undo data", "");
                 return;
             }
-            this._move(undoStates, redoStates);
-            this._restoreLastState();
+            // We want to know where we call this function: undo of workflow variables or tasks
```

Comment:
This comment is misleading, there are many different undo, not only workflow variables or tasks.

Reply:
that' right, I'll update it :)